packages/angular_devkit/build_angular/src/angular-cli-files/plugins/postcss-cli-resources.ts

```diff
@@ -117,7 +119,7 @@ export default postcss.plugin('postcss-cli-resources', (options: PostcssCliResou
 
         const outputPath = interpolateName(
           { resourcePath: result } as webpack.loader.LoaderContext,
-          filename,
+          path.join(assetsOutputPath, filename),
```

Comment:
do the join after the interpolate and use `path.posix.join`.  The option's value shouldn't be altered by the interpolation.